src/Keboola/StorageApi/TableExporter.php

```diff
@@ -69,7 +57,10 @@ private function handleExportedFile($tableId, $fileId, $destination, $exportOpti
             $fs->remove($destination);
         }
 
-        $tmpFilePath = $workingDir . '/' . uniqid('sapi-export-');
+        $downloader = DownloaderFactory::createDownloaderForPrepareRequest(
+            $fileInfo,
+            $this->client->getAwsRetries()
```

Comment:
Tady do toho trochu leakuje to AWSko, ale o moc líp to asi nejde. Maximálně kdyby tam byla DIC, tak by ta factory mohla být instance a mít to z DIC parametru přímo do konstruktoru...

Reply:
Ale protože je to klient, tak to nejde...